tests/services/contents/test_api.py

```diff
@@ -41,6 +42,13 @@ def dirs_only(dir_model):
 ]
 
 
+@pytest.fixture(params=["FileContentsManager", "AsyncFileContentsManager"])
+def argv(request):
+    if request.param == "AsyncFileContentsManager" and sys.version_info < (3, 6):
+        pytest.skip("Kernel manager is AsyncFileContentsManager, Python version < 3.6")
```

Comment:
Copy/paste - `Kernel` -> `Contents`.  Actually, since we no longer support 3.5, this condition can be dropped altogether. :smile: